ROADMAP-TO-1.0.0.md

```diff
@@ -438,7 +438,7 @@ The IPFS pub/sub service will provide fast delivery and routing for subnets of i
 - [ ] IPFS pub/sub: Protocol and architecture design
   - [ ] IPFS pub/sub: protocol spec / messages
   - [ ] IPFS pub/sub: record spec
-  - [ ] IPFS pub/sub: one-hop support
+  - [x] IPFS pub/sub: one-hop support
```

Comment:
We got the PubSubs and it gets used in multiple apps! (still needs that multihop tree forming stuffs)